mlir/cmake/modules/IRDLToCpp.cmake

```diff
@@ -0,0 +1,9 @@
+function(add_irdl_to_cpp_target target irdl_file)
+  add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${irdl_file}.cpp.inc
+    COMMAND ${MLIR_IRDL_TO_CPP_EXE} ${CMAKE_CURRENT_SOURCE_DIR}/${irdl_file} -o ${CMAKE_CURRENT_BINARY_DIR}/${irdl_file}.cpp.inc
+    DEPENDS ${MLIR_IRDL_TO_CPP_TARGET} ${irdl_file}
```

Comment:
Maybe you want `${CMAKE_CURRENT_SOURCE_DIR}/${irdl_file}` here or `${irdl_file}` above for consistency.